update getting started process (BREAKING CHANGES)

crhistianramirez opened the pull request:
## Description

- rename app settings so they're more clear
- add images to readme
- fix a few seeding bugs
- update seeding logic to use marketplace nomenclature

## BREAKING CHANGES

1. The /seed endpoint now takes in a `MarketplaceID` instead of a `SellerID` to better align with nomenclature referenced on our updated documentation.

2. The following app settings must be updated before pulling in this PR into your application:

| Before PR                               | After PR                                              |
|-----------------------------------------|-------------------------------------------------------|
| BlobSettings:ConnectionString           | StorageAccountSettings:ConnectionString               |
| BlobSettings:HostUrl                    | StorageAccountSettings:BlobPrimaryEndpoint            |
| BlobSettings:ContainerNameQueue         | StorageAccountSettings:BlobContainerNameQueue         |
| BlobSettings:ContainerNameExchangeRates | StorageAccountSettings:BlobContainerNameExchangeRates |
| BlobSettings:ContainerNameTranslations  | StorageAccountSettings:BlobContainerNameTranslations  |

maxwmaher commented on src/UI/Seller/src/assets/appConfigs/defaultadmin-test.json:
```diff
@@ -3,9 +3,9 @@
-  "middlewareUrl": "https://my-default-admin-test.com",
-  "translateBlobUrl": "https://my-translation-files/ngx-translate/i18n/",
-  "blobStorageUrl": "https://myblobstoragecontainer.blob.core.windows.net",
+  "clientID": "93268419-E864-444E-8BB1-1EB1B57C530B",
```
Confirming this should have a specific `clientID?`

crhistianramirez replied:
Good catch. Updated

Commit:
update getting started process

- rename app settings so they're more clear
- add images to readme
- fix a few seeding bugs
